tests/annotations_test.go

```diff
@@ -0,0 +1,44 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// ensures services.annotations values are applied to both coderd deployment &
+// service
+func TestAnnotations(t *testing.T) {
+	t.Parallel()
+
+	chart := LoadChart(t)
+
+	expected := map[string]string{}
```

Comment:
@jawnsy @deansheather in the prior commit, this read as:

```
expected := map[string]string{
    "key": "value",
    "service.beta.kubernetes.io/aws-load-balancer-protocol": "http",
}
```

I removed the `"key": "value"` requirement because the test was failing. Let me know if this is still needed. Test failure output was:

```
expected: map[string]string{"key":"value"}
actual  : map[string]string{}
```

Reply:
I replied here https://github.com/coder/enterprise-helm/pull/217#discussion_r799828357